Libs/MRML/Core/vtkMRMLModelDisplayNode.cxx

```diff
@@ -186,7 +186,7 @@ void vtkMRMLModelDisplayNode::SetThresholdRange(double min, double max)
 {
   vtkMTimeType mtime = this->ThresholdFilter->GetMTime();
   this->ThresholdFilter->ThresholdBetween(min, max);
-  if (mtime >= this->ThresholdFilter->GetMTime())
+  if (mtime >= this->ThresholdFilter->GetMTime() || !this->GetInputMesh())
```

Comment:
@lassoan indeed: done. I did the same for the AssignAttribute filter but it needed a couple more tweaks to work since the output data object of the filter was needed to find the data scalar range.